audiophiler/__init__.py

```diff
@@ -275,8 +277,10 @@ def toggle_tour_mode(auth_dict=None):
         return "Permisssion Denied", 403
 
     admin_query = Tour.query.first()
-    admin_query.tour_lock = not admin_query.tour_lock
-
+    if request.json["state"] == "t"
+        admin_query.tour_lock =  True
+    else if request.json["state"] == "f"
+        admin_query.tour_lock =  False
```

Comment:
Are these sure to be lowercase `t` and `f`? I'm pretty sure flask's json parsing does some boolean conversion you might look into.